.github/workflows/go-driver.yml

```diff
@@ -2,10 +2,10 @@ name: Go Driver Tests
 
 on:
   push:
-    branches: [ "master", "PG15" ]
+    branches: [ "master", "pg16" ]
```

Comment:
All `pg16 ` entries need to be uppercase `PG16`. Except where that `pg15` is lowercase.